### PR TITLE
CIP-0030 | A few small fixes

### PR DESCRIPTION
**Pull request by Ryun1**
See these few small formatting fixes.

#### Changes:
1. Changed _old style_ links that pointed to `CIP-0005.md`, these would require a short redirect because of past change to CIP markdown file naming.
2. Added javascript language indicators for code snippet data types and error codes.
3. Added missing `APIError` for `api.getExtensions()`.
4. Fixed incorrect reference to `cardano.{walletName}.extensions` which was removed during development of #446. Replaced with correct reference to `api.getExtensions()`.
5. Fixed highlighting for references to `APIError`, to all be `APIError` and not APIError.
6. A couple small spelling fixes.

#### Motivation:
1. Just more convenient to follow direct links.
2. Will rendered the snippets with syntax highlighting which improves readability.
3. This seems like an oversight and not intentional design.

### CODE REVIEW

**Review by rphair — APPROVED** [review]
Looks good to me & it's especially helpful to have that update from the `extensions` field to method: that could cause some real problems for new arrivals to the standard (we tend to miss things like that when we've been working with something for a while).

Also tagging a few more contributors to the CIP in case they might add anything else (apologies if I've missed anyone): @rooooooooob @alessandrokonrad @refi93 @vsubhuman

**Comment by Crypto2099** [comment]
Simple, mostly cosmetic and housekeeping changes, see no reason not to merge as-is.

File: CIP-0030/README.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ A hex-encoded string of the corresponding bytes.
...
+A hex-encoded string representing [CBOR](https://tools.ietf.org/html/rfc7049) corresponding to `T` defined via [CDDL](https://tools.ietf.org/html/rfc8610) either inside of the [Shelley Multi-asset binary spec](https://github.com/input-output-hk/cardano-ledger-specs/blob/0738804155245062f05e2f355fadd1d16f04cd56/shelley-ma/shelley-ma-test/cddl-files/shelley-ma.cddl) or, if not present there, from the [CIP-0008 signing spec](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0008/README.md).
 This representation was chosen when possible as it is consistent across the Cardano ecosystem and widely used by other tools, such as [cardano-serialization-lib](https://github.com/Emurgo/cardano-serialization-lib), which has support to encode every type in the binary spec as CBOR bytes.
 
 #### DataSignature
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:64]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 2]
```diff
@@ -81,15 +81,15 @@ transaction_unspent_output = [
 
 then we define
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:84]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 3]
```diff
@@ -81,15 +81,15 @@ transaction_unspent_output = [
...
 type TransactionUnspentOutput = cbor<transaction_unspent_output>
 ```
 
 This allows us to use the output for constructing new transactions using it as an output as the `transaction_output` in the Shelley Multi-asset CDDL does not contain enough information on its own to spend it.
 
 #### Paginate
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:92]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 4]
```diff
@@ -99,17 +99,17 @@ Used to specify optional pagination for some API calls. Limits results to {limit
 
 #### Extension
 
-An extension is an object with a single field `"cip"` that describe a CIP number extending the API (as a plain integer, without padding). For example:
+An extension is an object with a single field `"cip"` that describes a CIP number extending the API (as a plain integer, without padding). For example:
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:104]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 5]
```diff
@@ -99,17 +99,17 @@ Used to specify optional pagination for some API calls. Limits results to {limit
...
 { "cip": 30 }
 ```
 
 ### Error Types
 
 #### APIError
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:112]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 6]
```diff
@@ -129,7 +129,7 @@ APIError {
 
 #### DataSignError
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:132]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 7]
```diff
@@ -147,7 +147,7 @@ type DataSignError = {
 
 #### PaginateError
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:150]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 8]
```diff
@@ -156,7 +156,7 @@ type PaginateError = {|
 
 #### TxSendError
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:159]:
```suggestion
```ts
```

File: CIP-0030/README.md [hunk 9]
```diff
@@ -172,7 +172,7 @@ type TxSendError = {
 
 #### TxSignError
 
-```
+```js
```
Comment (KtorZ) [inline CIP-0030/README.md:175]:
```suggestion
```ts
```

**Review by KtorZ — APPROVED** [review]

**Pull request merged**